### DISCUSSION TITLE
"No meetings were found matching the selected criteria."

### DISCUSSION THREAD

**Discussion by Area82** [code4recovery/12-step-meeting-list#539] [Q&A] [answered]
Hi, 

Whenever I access our meetings page I get an error stating "No meetings were found matching the selected criteria". I don't have any filters specified and there are plenty of active meetings. Has anyone experienced this before?

URL: https://area82aa.org/meetings/

<img width="1294" alt="Screen Shot 2021-09-27 at 6 51 58 PM" src="https://user-images.githubusercontent.com/86302482/134990593-48a7f1b2-fba5-4cb9-8b59-d90499e0ace2.png">

**Comment by tech2serve** [comment]
Wow. Personally I have not. And stranger, I'm not seeing any errors or warnings in the console (which should have been the ideal place to start troubleshooting).

Without that, I'm thinking the best way to troubleshoot is to think about when it start; what got changed (either by an or the hosting provider). I would also try to look in the actual database to confirm meetings are still there. I assume you've looked at the meeting page in the Admin console and they're present?

Also try refreshing permalinks (Settings-->Permalink-->Save). And clearing the server cache (check with your hosting provider on how to do this).

It seems like jQuery simply isn't running, so perhaps you installed another jQuery helper plugin or something?

I'm really just grasping at straws here. 

Hopefully some other @code4recovery/tsml-contributors will chime in.

**Reply by Area82** [reply]
Thanks for your input. 

But yes, I started with the console and checked that the meetings were in the admin. I have not installed any new apps, only a few updates. I tried deactivating each app to see if one of them was causing this issue. I also tried refreshing the peramalinks and got nothing. Where would I find the meetings in my database? I tried looking but it does not seem obvious. 

Thanks!

**Comment by 97eighty** [comment]
@Area82 Try this query:

select * from wp_posts where post_type = "tsml_meeting"

**Comment by 97eighty** [comment, marked as answer]
@Area82 

I just looked at your page and the meetings are listing correctly.

**Reply by Area82** [reply]
That is very odd, I have not touched the site since last night...

**Comment by 97eighty** [comment]
A caching plugin or caching at the server level would display this behavior. If the blank page was cached it would display until the cache refreshed. If you have a caching system, perhaps it can be cleared?
